tests/integration/test_notebooks_python.py

```diff
@@ -82,3 +82,30 @@ def test_surprise_svd_integration(notebooks, size, expected_values):
     for key, value in expected_values.items():
         assert results[key] == pytest.approx(value, rel=TOL)
 
+
+@pytest.mark.integration
+@pytest.mark.parametrize(
+    "size, expected_values",
+    [
+        ("1m", dict(rmse=0.9555,
```

Comment:
is this guy fast enough to add 10M?

Reply:
the recommendation side is super slow, because of generating all pairs of user item interactions and writing them to disk, there is a utility within vw that will generate recommendations but that isn't pre-installed.